### DISCUSSION TITLE
Reembolso

### DISCUSSION THREAD

**Discussion by danivsonf** [aclap-dev/video-downloadhelper#84] [License & Purchase requests]
### What's your issue?

Comprei a assinatura hoje às 22:40 usando o email danivsonfranciel@gmail.com, e 15min depois enviei um email solicitando o reembolso porque o download das aulas que eu queria não estavam dando certo. Quando fui pesquisar para fazer o reembolso nao consegui descobrir por onde era, então enviei um email às 22:54 para sales@downloadhelper.net e outro email às 22:56 para support@downloadhelper.net. Com isso, gostaria de saber se o meu pedido será concedido ou não

### The 10 first digits (only 10) of your license number or email address

danivsonfr

### I confirm that I didn't not paste my full license number

- [X] I confirm

**Comment by jcfrog** [comment]
Hi,
I sent you an email. 
FYI you have one month to ask for refund.
